forge.config.ts

```diff
@@ -40,9 +40,16 @@ const config: ForgeConfig = {
         certificateFile: process.env.WINDOWS_CERTIFICATE_PATH,
         certificatePassword: process.env.WINDOWS_CERTIFICATE_PASSWORD,
       },
+      iconUrl:
+        'https://raw.githubusercontent.com/grafana/k6-studio/refs/heads/main/resources/icons/logo.ico',
```

Comment:
this wants an url and even if I tried to make it work with `file://` for using the local icon, it didn't seem to work well 😢